src/clusterfuzz/_internal/issue_management/issue_filer.py

```diff
@@ -479,8 +490,16 @@ def file_issue(testcase,
 
   recovered_exception = None
   try:
+    if issue_tracker.project == "google-buganizer":
```

Comment:
@jonathanmetzman Do you see any issues with adding logs with this kind of data?

Reply:
I think the TL;DR here is some of our fuzzers are finding bugs and trying to assign them to buganizer components where ClusterFuzz does not have access. So, it silently fails to create bugs in the component's where developers are expecting them.

Reply:
What do you think the logging issue would be?